src/transpose/index.js

```diff
@@ -4,6 +4,27 @@
  * @param {number[]} array The array to transpose
  * @returns {number[]} The transposed array
  */
-function transpose(array) {}
+function transpose(array) {
+	let tLength;
+	let m = array.length;
+	let n = array[0].length;
+	let transposing = [];
+	let transposed =[];
+	if ((1 <= m <= 10)&&(1 <= n <= 10)) {
```

Comment:
This conditional can be very confusing later on. Simplifying it would be better.